### ISSUE TITLE
austin-dist 3.7.0 upload arm64 build as x86-64 on PyPI

### ISSUE THREAD

**Issue by mrluanma** [P403n1x87/austin#242]
```
❯ uname -a
Darwin lua 24.0.0 Darwin Kernel Version 24.0.0: Tue Sep 24 23:36:30 PDT 2024; root:xnu-11215.1.12~1/RELEASE_X86_64 x86_64 i386 Darwin

~
❯ uv pip install --target t --system austin-dist
Using Python 3.13.0 environment at /usr/local/opt/python@3.13/Frameworks/Python.framework/Versions/3.13
Resolved 1 package in 2ms
Installed 1 package in 2ms
 + austin-dist==3.7.0

~
❯ tree t
t
├── austin_dist-3.7.0.dist-info
│   ├── INSTALLER
│   ├── METADATA
│   ├── RECORD
│   ├── REQUESTED
│   └── WHEEL
└── bin
    └── austin

3 directories, 6 files

~
❯ file t/bin/austin
t/bin/austin: Mach-O 64-bit arm64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|PIE>

~
❯ t/bin/austin
exec: Failed to execute process 't/bin/austin': Bad CPU type in executable.
```

**Comment by P403n1x87** [comment]
@mrluanma thanks for reporting this. I believe this has been caused by the macos runner running on arm64 now 🙁 
```
Current runner version: '2.320.0'
Operating System
  macOS
  14.7
  [2](https://github.com/P403n1x87/austin/actions/runs/11333751734/job/31518524498#step:1:2)3H124
Runner Image
  Image: macos-14-arm64
  Version: 20241007.259
  Included Software: https://github.com/actions/runner-images/blob/macos-14-arm64/20241007.259/images/macos/macos-14-arm64-Readme.md
  Image Release: https://github.com/actions/runner-images/releases/tag/macos-14-arm64%2F20241007.259
Runner Image Provisioner
  2.0.[3](https://github.com/P403n1x87/austin/actions/runs/11333751734/job/31518524498#step:1:3)84.1+6d6c56aa16f1b9c7dd7935df5d63980397e[4](https://github.com/P403n1x87/austin/actions/runs/11333751734/job/31518524498#step:1:4)4def
```
I'll see if I can get that wheel re-uploaded with the correct binary

**Comment by P403n1x87** [comment]
New wheel uploaded. @mrluanma could you please check that the issue is now resolved, please? 🙏

**Comment by mrluanma** [comment]
It's working!   And thanks for this great tool!

```
❯ uv pip install --target t --system --verbose austin-dist
DEBUG uv 0.4.22 (34be3af84 2024-10-15)
DEBUG Searching for default Python interpreter in system path
DEBUG Found `cpython-3.13.0-macos-x86_64-none` at `/usr/local/bin/python3` (search path)
Using Python 3.13.0 environment at /usr/local/opt/python@3.13/Frameworks/Python.framework/Versions/3.13
DEBUG Using `--target` directory at t
DEBUG Acquired lock for `t`
DEBUG At least one requirement is not satisfied: austin-dist
DEBUG Using request timeout of 30s
DEBUG Solving with installed Python version: 3.13.0
DEBUG Solving with target Python version: >=3.13.0
DEBUG Adding direct dependency: austin-dist*
DEBUG No cache entry for: https://pypi.org/simple/austin-dist/
DEBUG Searching for a compatible version of austin-dist (*)
DEBUG Selecting: austin-dist==3.7.0 [compatible] (austin_dist-3.7.0-1-py2.py3-none-macosx_11_0_x86_64.whl)
DEBUG No cache entry for: https://files.pythonhosted.org/packages/82/ec/66d49f755a3e0a980324d9f3acc43de5bf0ea87eea52fd826d87300c7d7c/austin_dist-3.7.0-1-py2.py3-none-macosx_11_0_x86_64.whl.metadata
DEBUG Tried 1 versions: austin-dist 1
DEBUG Split specific environment resolution took 1.168s
Resolved 1 package in 1.16s
DEBUG Identified uncached distribution: austin-dist==3.7.0
DEBUG No cache entry for: https://files.pythonhosted.org/packages/82/ec/66d49f755a3e0a980324d9f3acc43de5bf0ea87eea52fd826d87300c7d7c/austin_dist-3.7.0-1-py2.py3-none-macosx_11_0_x86_64.whl
Prepared 1 package in 354ms
Installed 1 package in 3ms
 + austin-dist==3.7.0
DEBUG Released lock at `t/.lock`

~
❯ file t/bin/austin
t/bin/austin: Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|PIE|HAS_TLV_DESCRIPTORS>

~
❯ t/bin/austin --version
austin 3.7.0
```
